<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 18px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Barbaro&quot;;</string>
    </property>
    <property name="text">
     <string>Cryptage d'un message</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>1131</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Saisir</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Donner le mot cle:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="motCle">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remp1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Remplir</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tab">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>651</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 14px;</string>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>4</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>5</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>1111</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Remplir le fichier chaine.txt</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Donner la chaine:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="chaine">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remp2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Remplir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_crypt">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Crypter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>60</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Resultat.txt</string>
     </property>
    </widget>
    <widget class="QListWidget" name="affChaine">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>431</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="affResult">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>120</y>
       <width>431</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1170</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
